doc/source/User_guide/mesh.rst

```diff
@@ -20,32 +20,108 @@ All mesh operations are conveniently listed within the mesh object:
   :width: 800
   :alt: Mesh object List
 
+Mesh in Icepak
+--------------
 
-Icepak has a different approach to the mesh operations.
-Those are managed through mesh regions and can be edited directly from the pyaedt object.
+Icepak employs a unique approach to mesh operations, managed through mesh regions, which can be edited directly from the pyaedt object.
+
+**Global Mesh Region**
+
+The global mesh region object is accessed using:
+
+.. code:: python
+
+    glob_msh = ipk.mesh.global_mesh_region
+
+This object allows for the editing of global mesh settings through its `settings` property, which functions like a dictionary.
+
+.. code:: python
+
+    glob_msh.settings["MeshRegionResolution"] = 3
+
+Available keys differ depending on whether manual or automatic settings are used. This can be adjusted using the `manual_settings` boolean property of the global mesh region object.
+
+To modify the global region dimensions, access the `global_region` property and modify its attributes.
+
+.. code:: python
+
+    glob_msh.global_region.positive_z_padding_type = "Absolute Offset"
+    glob_msh.global_region.positive_z_padding = "5 mm"
+
+To modify the properties of the Region object (for example the gas inside the region), access the `object` property of the `global_region` object:
 
 .. code:: python
 
-    icepak_b = Icepak()
+    glob_reg = glob_msh.global_region
+    glob_reg.object.material_name = "Carbon Monoxide"
+
+This is a pointer to the same object accessible from `ipk.modeler["Region"]`.
+
+The image below summarizes these three objects:
```

Comment:
```suggestion
This is a pointer to the same object accessible from ``ipk.modeler["Region"]``.

The following image summarizes these three objects:
```
Avoid positional words like above and below in case the reader is looking a PDF or printed pages.